packages/react-core/src/components/Wizard/WizardToggle.tsx

```diff
@@ -52,10 +52,11 @@ export const WizardToggle: React.FunctionComponent<WizardToggleProps> = ({
     }
   }
   return (
-    <>
+    <React.Fragment>
       <button
         onClick={() => onNavToggle(!isNavOpen)}
         className={css(styles.wizardToggle, isNavOpen && 'pf-m-expanded')}
+        aria-label="Wizard toggle"
```

Comment:
no prop for this?

Reply:
Good catch, I added it to pass our a11y-tests which were previously not picking it up.